hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemFlush.java

```diff
@@ -418,7 +418,7 @@ private void validate(FileSystem fs, Path path, byte[] writeBuffer, boolean isEq
         assertArrayEquals(
                 String.format("Bytes read do not match bytes written to %1$s", filePath), writeBuffer, readBuffer);
       } else {
-        assertThat(
+          org.hamcrest.MatcherAssert.assertThat(
```

Comment:
Indentation here looks off, I think we want to keep the original.

Reply:
Hi, @dannycjones  ,Thank you very much for your help reviewing the code, I will fix it.